src/test/java/cloud/eppo/BaseEppoClientBanditTest.java

```diff
@@ -66,14 +75,16 @@ public static void initClient() {
             false,
             false,
             true,
-            null);
+            null,
+            new AbstractAssignmentCache(assignmentCache) {},
+            new ExpiringInMemoryAssignmentCache(banditAssignmentCache, 10, TimeUnit.MINUTES) {});
```

Comment:
🔥 

I think for the test we should make it something like 50ms, then have the test sleep 50 MS, and make sure it logs the delayed duplicate assignment